packages/breadcrumbs/src/containers/BreadcrumbContainer.js

```diff
@@ -5,6 +5,16 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
+if (process.env.NODE_ENV !== 'production' && process.env.NODE_ENV !== 'test') {
```

Comment:
Even though this would be noisy in our environment, I think removing `process.env.NODE_ENV !== 'test'` would be helpful.

Consumers should see these warnings in their tests if they're using a deprecated package.